data/campaigns/Eastern_Invasion/scenarios/05_Northern_Outpost.cfg

```diff
@@ -254,7 +254,16 @@
 
         {DACYN_CASTING_SPELL_ANIMATION_OBJECT}
 
-        {MODIFY_UNIT id=Dacyn variables.casting_the_spell yes}
+        [modify_unit]
+            [filter]
+                id=Dacyn
+            [/filter]
+
+            [set_variable]
+                name=casting_the_spell
+                value=yes
+            [/set_variable]
+        [/modify_unit]
     [/event]
```

Comment:
Unless we're going to deprecate the macro, I think the three changes in this file shouldn't be done - leaving them as one-liners is easier to read.